packages/fx-core/resource/package.nls.json

```diff
@@ -1021,6 +1021,7 @@
   "driver.botFramework.progressBar.createOrUpdateBot": "Creating or updating bot registration.",
   "driver.generateAppsettings.description": "Add or update environment variables to appsettings file.",
   "error.yaml.InvalidYamlSchemaError": "Unable to parse yaml file: %s",
+  "error.yaml.InvalidYamlSchemaErrorWithReason": "Unable to parse yaml file: %s. Reason: ",
```

Comment:
after "Reason" there is no any string for it, is it supposed to be like this?

Reply:
Nice catch. if there is no %s, the extra parameter string will be appended to the end.
![image](https://user-images.githubusercontent.com/18023393/229016448-b155c5dd-60c3-40c4-bfab-f3c9c7e5e3d9.png)

That's why I missed it.